app/models/paper_petition.rb

```diff
@@ -134,6 +134,8 @@ def save
     @signature = @petition.build_creator(signature_params)
     @contact = @signature.build_contact(contact_params)
 
+    @petition.create_pe_number
```

Comment:
This should be `build_pe_number` so that it all occurs in a single transaction

Reply:
I have a feeling that that did not work for some reason, it didn't save it?